### DISCUSSION TITLE
myBMW app connection not working

### DISCUSSION THREAD

**Discussion by generalks** [BimmerGestalt/AAIdrive#392] [Q&A]
Hi,

I'm running NBT / id4 and am experiencing some issues with the myBMW app. The behaviour I am seeing is that after connecting with USB the app will launch and the notification will display on my phone saying it is connected. The car connection assistant will sit on 'Apps loading...' for a few moments before displaying the 'Downloaded BMW Connected app' message...

In the AAI app, the tunnel status seems to cycle constantly between connected/established and disconnected. When up it shows 4 connections and seems to last for 10 seconds or so before disconnecting again.

So far I have only seen the apps load successful once and have not been able to recreate it. Has anyone else experienced this issue?

Cheers.

**Comment by hufman** [comment]
I saw a [recent forum post](https://www.bimmerfest.com/threads/unofficial-android-app-to-add-more-music-apps-and-phone-messages-via-bmw-connected.1341733/page-9#post-13591520) indicating that recent MyBMW versions might be flaky, and to downgrade to version 1.5.0 for better luck, maybe that will help you too?

**Comment by matztam** [comment]
Same here (NBT / ID4 / Android 11). It works when I disable Bluetooth before connecting the usb cable. When the apps are loaded Bluetooth can be reenabled.
Alternative: [Connected Classic](https://apkpure.com/p/com.bmwgroup.connected.bmw) app works with Bluetooth enabled.

**Reply by drm87** [reply]
Running the same config (ID4, Android 11 (Pixel 5)) and the current MyBMW app (1.5.2) and everything works flawlessly, no need to enable or disable Bluetooth or something like that.
I just connect the phone via USB and do not have to do anything manually.

Are you running the latest nightly AAIdrive app version or the latest release version?

**Reply by matztam** [reply]
I'm running the latest stable release (version 1.3)

**Comment by unlive74** [comment]
all MyBMW builds older than 1.4.3 connect to my car (x2, f39, id6L) without apps until i run MyBMW app on my samsung s8, stock (even if it started an hour ago).  Battery saver for app is disabled in android settings.  App build 1.4.3 always connects correctly.

**Comment by Kay0002** [comment]
Hi @hufman, as of today I cannot use the app. iDrive no longer has the App Connection checkbook under my Bluetooth devices. I'm using iDrive 6.

I'm assuming it's an issue with the iDrive software, meaning I'll have to go to service, but was wondering if you had any thoughts. 

![20220406_132631](https://user-images.githubusercontent.com/77034203/162036371-eceed6ad-c4d2-4e4f-a2a3-d486a5cabc04.jpg)

**Reply by hufman** [reply]
It sounds like the car forgot about your BMW ConnectedDrive subscription. Check that the subscription is active in MyBMW, and then follow [these instructions](https://www.reddit.com/r/BMW/comments/f8fn6n/comment/fkkwdsn/?utm_source=share&utm_medium=web2x&context=3) to force the car to update the subscription status (which otherwise takes a day or two to check in the background)

**Reply by Kay0002** [reply]
That was it, thanks for the help 🙂
